test/classes/delete-free/lifetimes/borrow-escapes.chpl

```diff
@@ -9,14 +9,14 @@ class MyClass {
 
 record R {
   // TODO - get init with owned fields working
-  var c:owned MyClass;// = new Owned(nil:MyClass);
+  var c:owned MyClass?;// = new Owned(nil:MyClass);
   proc init() {
     //var tmp = new Owned(new MyClass(data));
     //c = tmp;
     //c.retain(new MyClass(data));
   }
   proc get(): borrowed MyClass {
-    return c.borrow();
+    return c!.borrow();
```

Comment:
Please make an issue about this

Reply:
@mppf what do you think the issue is here?

Here is what I am seeing:

* If I remove the `!` added above, I get a misleading error message. I will create an issue about that. However, `!` **can** be removed. The test passes without the `!` -- as long as the return type of this `get()` is switched to nilable, and so are the return types of the functions that are affected by this.

* So I would like to adjust this test by at least removing the `!`. My question is: is the intention that the field R.c be nilable? Or should it be set to `new MyClass(something)` upon initializing an R variable? The test passes if I do the latter.

Reply:
P.S. For R.c to be non-nilable, init() can be empty, we can just write:
```chpl
var c = new owned MyClass(55);
```

Or is the commented-out code in R.init() intended to be there? We can probably uncomment it now instead of the above.

Reply:
That test doesn't have anything to do with nilibility. It would be fine to adjust `c` to always initialize with a new class. This ought not cause different errors for the lifetime checking, which is what the test checks.